src/DatePicker.elm

```diff
@@ -82,7 +83,6 @@ type Msg
     = NoOp
     | DateSelected Date Date
     | GetToday Date
-    | SetYear Date Int
```

Comment:
This was not being used. `DateSelected` seemed completely fine for updating the year.